readme.md

```diff
@@ -111,6 +111,79 @@ python3 -m nwm_routing -f -V4 test_AnA_V4_HYFeature.yaml
 
 **Note**: The following instructions are for setting up T-Route on a Linux environment (standalone, no MPI). If you are using Windows, please install WSL (Windows Subsystem for Linux) before proceeding.
 
+### T-Route Setup and Testing Guide for Windows Users WITHOUT conda [based on pip and venv - only widely available dependencies]
+
+1. **Install Recommended distro:**
+   - Download and install WSL2 for your Windows OS
+   - We recommend long-term stable (LTS) Ubuntu distribution 22.04 or 20.04 (24.04 not recommended yet)
+   - Open Windows Power Shell and issue
+     > wsl --install Ubuntu-22.04
```

Comment:
If we specify the Version for Ubuntu, soon it will be outdated and we have to keep updating this. Its better to ask user, to find the latest stable version and install that.

Reply:
Amin: Ubuntu delivers a "long-term-stable" version of its Linux distro every two years. At the moment, there are 18.04, 20.04, 22.04, and 24.04 in active usage, referring to the release years. As we are developing scientific modeling software, rather than systems software, we should stick to these long-term stable distros as long as they are in wide use, and as long as they work with one of the corresponding standard Python versions. 

T-route's standard tests on github cover Python 3.9, 3.10, and 3.11. Ubuntu 20.04 and Ubuntu 22.04 come with Python 3.9 and 3.10, respectively, whereas Ubuntu 24.04 has Python 3.12. So, we should definitely NOT recommend Ubuntu 24.04 yet, because there may be compatibility issues, until such time that T-route is released for Python 3.12. Hence, Ubuntu 22.04 is the most recent Ubuntu distro that is consistent with T-route. I do think that, for the same rationale that we set Python library versions in the requirements.txt file, we not only can but should make sure to recommend a Ubuntu version that works with T-route on github. 

Given that 20.04 and even 18.04 are still widely used, 22.04 has another 2-4 years of runway - I think that is manageable in terms of durability.

Reply:
Yes. Make sense!